tutorials/using-xgboost/environment.yml

```diff
@@ -0,0 +1,16 @@
+channels:
+  - conda-forge
+dependencies:
+  - python=3.8
+  - pip
+  - pip:
+    - https://default7955323798.blob.core.windows.net/publictestwheels/azureml_core-0.1.0.0-py3-none-any.whl # azureml-core
```

Comment:
Should point to packages in PyPI, not private blobs.

Reply:
agree, waiting on public release, will not merge until changed

Reply:
[internal teams thread](https://teams.microsoft.com/l/message/19:c9f97cad7a7746789df07ebf82436fc3@thread.skype/1610403060948?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=88aa174e-6310-4634-bfcb-5761e1a1190a&parentMessageId=1610403060948&teamName=ML%20Platform&channelName=ASK%20MODE&createdTime=1610403060948)